.travis.yml

```diff
@@ -14,14 +14,12 @@ env:
 matrix:
     fast_finish: true
     include:
-        - php: 7.1
```

Comment:
I'm not sure removing the tests for 7.1 is the best thing to do here. As the bundle it self still supports 7.1, we _should_ also test against 7.1. I do however understand the conflict between 7.1 & PhpUnit 8. That said, I'll dig around how we handle this in the our other repo's. There is a solution - I just cant think of it off the top of my head.

Reply:
:thinking: It probably works by setting the `SYMFONY_PHPUNIT_VERSION` [env var](SYMFONY_PHPUNIT_VERSION) in the travis config. 

I haven't done this in any project before and would appreciate a working example :slightly_smiling_face:. Travis is taking very long to pick up jobs these days which makes trial and error a bit cumbersome.

Reply:
no problem - see my suggestion below, then keeping `- php: 7.1`; all tests should pass without problems.

Reply:
see 895364cd7a265814b496321fa6b8a5b95d008e88